doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -0,0 +1,89 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+from urllib import request
+import json
+from packaging.version import Version
+
+JUMPSTART_REGION = "eu-west-2"
+SDK_MANIFEST_FILE = "models_manifest.json"
+JUMPSTART_BUCKET_BASE_URL = "https://jumpstart-cache-prod-{}.s3.{}.amazonaws.com".format(
+    JUMPSTART_REGION, JUMPSTART_REGION
+)
+
+
+def get_jumpstart_sdk_manifest():
+    url = "{}/{}".format(JUMPSTART_BUCKET_BASE_URL, SDK_MANIFEST_FILE)
+    with request.urlopen(url) as f:
+        models_manifest = f.read().decode("utf-8")
+    return json.loads(models_manifest)
+
+
+def get_jumpstart_sdk_spec(key):
+    url = "{}/{}".format(JUMPSTART_BUCKET_BASE_URL, key)
+    with request.urlopen(url) as f:
+        model_spec = f.read().decode("utf-8")
+    return json.loads(model_spec)
+
+
+def create_jumpstart_model_table():
+    sdk_manifest = get_jumpstart_sdk_manifest()
+    sdk_manifest_top_versions_for_models = {}
+
+    for model in sdk_manifest:
```

Comment:
Future improvement (non-blocking):
Do we know how much time is added to read and update read the docs for all the jumpstart models (I heard no of models are 300+)?
If it takes a significant time, alternatively we could have checked for jumpstart manifest hash (md5 or something) and compared against one stored in sagemaker defalt bucket and only update if hash mismatches.

Reply:
I've seen this take a max of 5 minutes as it stands. I agree we should definitely look to optimize this call, but we wanted to get `P0` into production ASAP so customers could access the model table. We can discuss the implementation further at our next sync.